neurom/features/neuronfunc.py

```diff
@@ -244,18 +244,18 @@ def _count_crossings(neurite, radius):
         return count
 
     return np.array([sum(_count_crossings(neurite, r)
-                         for neurite in iter_neurites(neurites))
+                         for neurite in iter_neurites(neurites, filt=neurite_filter))
                      for r in radii])
 
 
 @feature(shape=(...,))
-def sholl_frequency(nrn, neurite_type=NeuriteType.all, step_size=10):
+def sholl_frequency(nrn, neurite_type=NeuriteType.all, bins=None):
     """Perform Sholl frequency calculations on a population of neurites.
 
     Args:
         nrn(morph): nrn or population
         neurite_type(NeuriteType): which neurites to operate on
-        step_size(float): step size between Sholl radii
+        bins(iterable of floats|int|None): binning to use for the Sholl radii
```

Comment:
@arnaudon can you please add more documentation to this parameter?

Reply:
ah sorry, I just saw you did it, it's good to me, just the thing with bins=None I find a bit weird, but as you prefer

Reply:
done